.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,67 @@
+# Please reference work here https://github.com/EVerest/everest-core/tree/main/.github/workflows
+# TODO: modify to reuse the above workflow to DRY up CI.
+
+name: Build and test liblog
+on:  
+  push:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
```

Comment:
Please have a look at https://github.com/EVerest/everest-core/blob/main/.github/workflows/build_and_test.yaml#L20 and use the defined input